processing/src/test/java/io/druid/guice/TestTableModule.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.guice;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Binder;
+import com.google.inject.Module;
+import com.google.inject.Provides;
+import io.druid.jackson.DefaultObjectMapper;
+import io.druid.metadata.MetadataStorageTablesConfig;
+
+import java.util.Arrays;
+
+public class TestTableModule implements Module
+{
+  @Override
+  public void configure(Binder binder)
+  {
+    binder.install(new PropertiesModule(Arrays.asList("test.runtime.properties")));
+    binder.install(new ConfigModule());
+    binder.install(new DruidGuiceExtensions());
+    JsonConfigProvider.bind(binder, "druid.metadata.storage.tables", MetadataStorageTablesConfig.class);
+  }
+  
+  @Provides @LazySingleton
+  public ObjectMapper jsonMapper()
```

Comment:
this is probably not needed and if it can be removed than all that is left is the configure(..). given that this class is doing so little, I think it can be moved to MetadataStorageTablesConfigTest.java as an anonymous class.